modules/nw-ovn-kubernetes-logs-cli.adoc

```diff
@@ -36,32 +36,32 @@ For example:
 +
 [source,terminal]
 ----
-$ oc logs ovnkube-master-7h4q7 -n openshift-ovn-kubernetes
+$ oc logs ovnkube-node-5dx44 -n openshift-ovn-kubernetes
 ----
 +
 [source,terminal]
 ----
-$ oc logs -f ovnkube-master-7h4q7 -n openshift-ovn-kubernetes -c ovn-dbchecker
+$ oc logs -f ovnkube-node-5dx44 -c ovn-dbchecker -n openshift-ovn-kubernetes
```

Comment:
I could not get this command to work should this container be `ovnkube-controller`? Or is this command not relevant for 4.14+? @tssurya @anuragthehatter

Reply:
The error says that the `-c` `ovn-dbchecker` is not valid for <pod_name>

Reply:
`ovn-dbchecker` no more exist in IC. we can reference any other container here.

Reply:
Tested on live cluster and discussed with @anuragthehatter changing to `oc logs -f ovnkube-node-78gmg -c ovnkube-controller -n openshift-ovn-kubernetes`